k8s/live/azure-dev/kustomization.yaml

```
@@ -8,6 +8,16 @@ components:
   - ../../components/gridsuite
   - ../../components/azure
 
+resources:
```

yamllint (1.35.1)

[error] duplication of key "resources" in mapping

(key-duplicates)